tests/e2e/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "devDependencies": {},
   "scripts": {
-    "test": "jest --runInBand --colors",
+    "test": "jest --debug --runInBand --colors",
```

Comment:
hmm this will also be run by the e2e CIs. are you sure about this?

Reply:
I think so, it will just print some information about the parameters used for testing. The rest seems to be the same.